app/models/validators/placement_zones_validator.rb

```diff
@@ -0,0 +1,16 @@
+class PlacementZonesValidator < ActiveModel::EachValidator
+  ZONE_TYPES = %w(o a b i)
```

Comment:
Not sure if this is a good or bad idea.  The field is called `reject_zones` ... so presumably we'd reject any zones with these types?  This doesn't give a way to differentiate between house and paid ad zones at the moment though.

Guess we can figure that out later.